live/root/usr/lib/dracut/modules.d/99agama-cmdline/agama-cmdline-conf.sh

```diff
@@ -15,7 +15,7 @@ get_agama_args() {
       _found=1
       _env=1
       ;;
-    agama*)
+    inst* | agama*)
```

Comment:
Maybe we should inform that the agama*whatever parameter is deprecated in favor of the inst equivalent... like the getarg -d option...

Reply:
```agama``` being obsolete is covered in documentation update ...

question is whether will someone be able to recognize such output ?